### DISCUSSION TITLE
The default language is set to "Simplified Chinese", why is the menu still in English? Many places are in English.

### DISCUSSION THREAD

**Discussion by qq1056123185** [pawelsalawa/sqlitestudio#4038] [Q&A]
The default language is set to "Simplified Chinese", why is the menu still in English? Many places are in English.

**Comment by pawelsalawa** [comment]
It's because messages were not entirely translated to Chinese. At least in 3.3.x version.

The next major update (3.4.0) will contain much more updated messages (including Chinese), as people are contributing translations quite a lot there (for version 3.3.x it was not as easy).
